tools/config_includer/generator.go

```diff
@@ -121,7 +121,7 @@ func main() {
 			pwd = string(filepath.Separator) + pwd
 			for _, iname := range extractIncludes(tmpl) {
 				ifn := iname
-				if !filepath.IsAbs(ifn) {
+				if !strings.HasPrefix(ifn, "/") {
 					ifn = filepath.Join(pwd, ifn)
```

Comment:
Your change is good but I wonder if the line below should read
```suggestion
					ifn = filepath.Join(pwd, filepath.FromSlash(ifn))
```

Reply:
Windows doesn't mind slashes as path separators, but filepath.Join already turns them into backslashes anyway.

Reply:
Cool, didn't know...